.ci/pytorch/windows/arm64/bootstrap_buildtools.bat

```diff
@@ -1,4 +1,5 @@
 @echo off
+setlocal enabledelayedexpansion
```

Comment:
What was the issue with the original version of the script so this line had to be added?